### PR TITLE
Implement feedback on changelog display

### PR DESCRIPTION
**Pull request by Tehsurfer**
# Description

Added a few details to the changelog to make it clearer
image
https://www.wrike.com/open.htm?id=947150591

## Type of change

Delete those that don't apply.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# How Has This Been Tested?

Tested locally, will have herokuapp here soon:
https://jesse-sprint-23.herokuapp.com/maps

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have utilized components from the Design System Library where applicable
- [ ] I have added unit tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: components/VersionHistory/VersionHistory.vue [hunk 1]
```diff
@@ -1,21 +1,23 @@
 <template>
   <div class="version-history-container">
     <large-modal
+      v-if="dialogVisible"
       :visible="dialogVisible"
```
Comment (hsorby) [inline components/VersionHistory/VersionHistory.vue:5]: Do you need both of these?

Comment (Tehsurfer) [reply components/VersionHistory/VersionHistory.vue:5]: @hsorby  yes, some of the contents in the large-modal are not available onMount. 

using v-if allows them to only be loaded just in time.

the `:visible` one is required for the modal so we end up needing both

Comment (Tehsurfer) [reply components/VersionHistory/VersionHistory.vue:5]: I could remove the `v-if`
I would put all of the logic in `computed` properties to check if the object parameters exist or not.
ie:
```
  <p>Published on: {{logPublishedAt}}</p>

//...
  computed: {
    logPublishedAt: function(){
      if( this.changeLogFileInfo.versionPublishedAt) {
        return formatDate(this.changeLogFileInfo.versionPublishedAt)
      }
      return ''
    }
```


I think the `v-if` is cleaner personally, but I could use your feedback so I can get this in before the weekend

Comment (hsorby) [reply components/VersionHistory/VersionHistory.vue:5]: Using a computed property is the way that I would do it personally. I think that is a very Vue way of handling this problem.

Comment (Tehsurfer) [reply components/VersionHistory/VersionHistory.vue]: @hsorby I have switched to the way you would like to do it. All good for approval now?

**Review by hsorby — APPROVED** [review]

**Pull request merged**